runtime/libpgmath/tools/tmp-mth_statsdefs_x86_64.ps1

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
```

Comment:
Why is this using the old license?

Reply:
Isn't this the new license and the others are the old license?

Reply:
Yes, you are right. This is the new license. Ignore my comment for this one.